ndmg/stats/qa_tractography.py

```diff
@@ -0,0 +1,113 @@
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+
+from fury import actor, window, colormap, ui
+from dipy.tracking.streamline import Streamlines
+from PIL import Image
+from matplotlib.colors import LinearSegmentedColormap
+
+def combine_plot(qa_out_path,brain_path):
+    """
+    Combine the nine plots produced by qa_tractography function into one image.
+
+    Parameters
+    ----------
+    qa_out_path : str
+        Path for the output QA imgae file(s)
+    brain_path : str
+        Path for the reference brain scan, in order to get the scan volume.
+    """
+    
+    brain_ref = nib.load(brain_path)
+    brain_ref_shape = brain_ref.shape[0:3]
+
+#   plt.rcParams.update({"axes.labelsize":"x-large","axes.titlesize":"x-large"})
+    plots_list = ['imgage1000000.png','imgage1000001.png','imgage1000002.png',
+                 'imgage2000000.png','imgage2000001.png','imgage2000002.png',
+                 'imgage3000000.png','imgage3000001.png','imgage3000002.png']
+    y_labels = ["Rotate with Z axis","Rotate with Y axis","Rotate with X axis"]
+    x_labels = ["angle=0","angle=120","angle=240"]
+    
+    #Temporary list to store the open image:
+    im=[] 
+    #Temporary list to store the image nparray:
+    im_arr=[] 
+
+    fig= plt.figure()
+    plt.title(f'QA_tractography. Scan volume = {brain_ref_shape} \n\n', fontsize=60,fontweight='bold')
+    plt.xticks([])
+    plt.yticks([])
+    plt.axis("off")
+
+    j = 0
+    for i in range(9):
+        im.append(Image.open(qa_out_path + "/"+ plots_list[i]))
+        im_arr.append(np.array(im[i])) 
```

Comment:
You append to `im` and `im_arr` while only using `im_arr[i]` in the code below. I would replace this with `im_arr = np.array(Image.open(qa_out_path + "/" + plots_list[i]` and then use `im_arr` instead of `im_arr[i]` in the code below.

Reply:
https://github.com/neurodata/ndmg/pull/375/commits/f9edd4a64206da4025ac3ece5970c649b197ff73